modules/docker/docker_compose.go

```diff
@@ -16,15 +17,26 @@ type Options struct {
 
 // RunDockerCompose runs docker-compose with the given arguments and options and return stdout/stderr.
 func RunDockerCompose(t testing.TestingT, options *Options, args ...string) string {
-	out, err := RunDockerComposeE(t, options, args...)
+	out, err := runDockerComposeE(t, false, options, args...)
 	if err != nil {
 		t.Fatal(err)
 	}
 	return out
 }
 
+// RunDockerComposeAndGetStdout runs docker-compose with the given arguments and options and returns only stdout.
+func RunDockerComposeAndGetStdOut(t testing.TestingT, options *Options, args ...string) string {
+	out, err := runDockerComposeE(t, true, options, args...)
+	require.NoError(t, err)
+	return out
+}
```

Comment:
Just realized that we should probably expose a `RunDockerComposeAndGetStdOutE` function as well.